helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/daemonset.yaml

```diff
@@ -60,6 +60,7 @@ spec:
       {{- end }}
       containers:
       - name: splunk-fluentd-k8s-logs
+        command: ["/bin/sh", "-c", "bundle exec fluentd -c /fluentd/etc/fluent.conf"]
```

Comment:
can you add a comment on how this is derived? e.g. a link to the dockerfile you looked up.

I also suggest on using comments (if supported) to indicate which parts are overwriten by us

Reply:
If we want to make it obvious, how about we add a changelog then?

Reply:
I slightly prefer adding context in `README.md` and adding something standing out (e.g. `####opendoor override####`) in comments here for quickly identify the content (similar to how git merge conflicts are marked)